### DISCUSSION TITLE
Cannot email patch

### DISCUSSION THREAD

**Discussion by ryspace** [GothenburgBitFactory/taskwarrior#3025] [Community support]
I sent in a patch on the 15th - ironically updating the patch submission documentation, and just received a return notice last night:

```
[<support@gothenburgbitfactory.org>](mailto:support@gothenburgbitfactory.org): host mx2.forwardmx.io[209.151.145.137]
    said: 451 The domain gothenburgbitfactory.org is currently suspended. Try
    later. (in reply to RCPT TO command)
```

It would appear that no one from the team is actually in the matrix/libera.chat room, and (even though I don't personally have an account) the Twitter account hasn't been updated in over a year, so this appears to be the only way to contact anyone...

**Comment by djmitche** [comment]
Indeed, I believe GH pull requests are the currently-accepted way to make contributions.

There was a period of broadening of the communication tools -- matrix, discussions, twitter, discord, etc.  But the breadth is just too much for the limited bandwidth of the developers, so we're largely limited to this GH repo at the moment.  It probably makes sense to update some docs that currently point to old, un-maintained communication tools.

**Comment by ryspace** [comment]
Okay, I'll fork and redo it then...that said, that email is listed all over GBF's website as the only means to contact them in many respects, so it might be something to look into that it's not working.

**Reply by tbabej** [reply]
Thanks for notifying us about this, we'll look into reviving the email. Though as Dustin said, the Github repo is likely most active.
